FileIO/writeGeometryToFile.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "writeGeometryToFile.h"
+
+#include "BaseLib/FileTools.h"
+
+#include "FileIO/XmlIO/Boost/BoostXmlGmlInterface.h"
+#include "FileIO/Legacy/OGSIOVer4.h"
+
+#include "GeoLib/GEOObjects.h"
+
+namespace FileIO
+{
+void writeGeometryToFile(std::string const& geo_name,
+	GeoLib::GEOObjects& geo_objs, std::string const& fname)
+{
+	if (BaseLib::getFileExtension(fname).compare("gml") == 0) {
+		FileIO::BoostXmlGmlInterface xml(geo_objs);
+		xml.setNameForExport(geo_name);
+		xml.writeToFile(fname);
+	} else {
```

Comment:
Maybe also compare to "gli" and print an error message if none of the options match?